### ISSUE TITLE
Not reading my touch, when pressing start new wall.

### ISSUE THREAD

**Issue by SebastainMisas** [bjarnel/arkit-occlusion#1]
Hello, I downloaded this repo and tried running it on my iphone device. Looks like when i press the new wall button it stays at a blank camera screen and doesn't do anything when i touch to set the new wall.

**Comment by bjarnel** [comment]
Hi Sebastain,
Hmm, a few things 1) you need to move the device around the room for a bit, 2) the wall/surface you want to place wall on has to have some degree of texture.
My demo apps currently does not show any tracking status info :/
Probably I should add more info :-)

**Comment by wave-electron** [comment]
Yeah adding status would be a big help in troubleshooting. I found it really struggles with white walls. But is still a great demo of what’s possible by adding wall & object collisions.  Could see people having fun with a bowling pin or basketball shoot app.
